examples/webgl_effects_ascii.html

```diff
@@ -5,6 +5,9 @@
 		<meta charset="utf-8">
 		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
 		<link type="text/css" rel="stylesheet" href="main.css">
+		<style>
+			body { overflow: hidden; }
+		</style>
```

Comment:
@Mugen87 Why is this CSS needed here, when it is not needed in the other 340 examples?